apps/cowswap-frontend/src/legacy/theme/baseTheme.tsx

```diff
@@ -400,6 +400,18 @@ export const UniThemedGlobalStyle = css`
     --cow-color-yellow-light: ${({ theme }) => theme.alert2};
     --cow-color-green: ${({ theme }) => theme.success};
 
+    // Base
+    --cow-color-border: var(--cow-color-grey);
+    --cow-container-bg-01: ${({ theme }) => theme.bg1};
+    --cow-modal-backdrop: var(--cow-color-text1-opacity-25);
+    --cow-border-radius-normal: 24px;
+    --cow-padding-normal: 24px;
+    --cow-box-shadow-normal: 0 1.5rem 1rem #00000008,0 .75rem .75rem #0000000a,0 .25rem .375rem #0000000d;
```

Comment:
Here is a base place for their definition, using the same way.

It's just a template string, so it shouldn't add any extra processing/make it slower

```suggestion
    ${ColorTypes.COLOR_BORDER}: var(${ColorTypes.COLOR_GREY});
    --cow-container-bg-01: ${({ theme }) => theme.bg1};
    --cow-modal-backdrop: var(--cow-color-text1-opacity-25);
    --cow-border-radius-normal: 24px;
    --cow-padding-normal: 24px;
    --cow-box-shadow-normal: 0 1.5rem 1rem #00000008,0 .75rem .75rem #0000000a,0 .25rem .375rem #0000000d;
```